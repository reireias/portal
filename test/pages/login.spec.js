import { mount, createLocalVue } from '@vue/test-utils'
import Vue from 'vue'
import Vuetify from 'vuetify'
import tartget from '@/pages/login.vue'

const localVue = createLocalVue()

Vue.use(Vuetify)
localVue.use(Vuetify)

describe('login', () => {
  test('is a Vue instance', () => {
    const vuetify = new Vuetify()
    const wrapper = mount(tartget, {
      localVue,
      vuetify,
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
