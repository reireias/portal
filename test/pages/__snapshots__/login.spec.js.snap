// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index is a Vue instance 1`] = `
"<div class=\\"container\\">
  <!---->
  <div class=\\"row justify-center\\">
    <div class=\\"col-md-6 col-12\\">
      <div class=\\"v-input theme--light v-text-field\\">
        <div class=\\"v-input__control\\">
          <div class=\\"v-input__slot\\">
            <div class=\\"v-text-field__slot\\"><label for=\\"input-3\\" class=\\"v-label theme--light\\" style=\\"left: 0px; position: absolute;\\">Email</label><input id=\\"input-3\\" type=\\"text\\"></div>
          </div>
          <div class=\\"v-text-field__details\\">
            <div class=\\"v-messages theme--light\\">
              <div class=\\"v-messages__wrapper\\"></div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div class=\\"row justify-center\\">
    <div class=\\"col-md-6 col-12\\">
      <div class=\\"v-input theme--light v-text-field\\">
        <div class=\\"v-input__control\\">
          <div class=\\"v-input__slot\\">
            <div class=\\"v-text-field__slot\\"><label for=\\"input-6\\" class=\\"v-label theme--light\\" style=\\"left: 0px; position: absolute;\\">Password</label><input id=\\"input-6\\" type=\\"password\\"></div>
          </div>
          <div class=\\"v-text-field__details\\">
            <div class=\\"v-messages theme--light\\">
              <div class=\\"v-messages__wrapper\\"></div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div class=\\"row justify-center\\"><button type=\\"button\\" class=\\"v-btn v-btn--contained theme--light v-size--default\\"><span class=\\"v-btn__content\\">login</span></button></div>
</div>"
`;
